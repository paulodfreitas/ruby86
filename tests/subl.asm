.module entry_code
.pseg
.org 0
        irmovl $3, %eax
        irmovl $4, %ebx
        subl %eax, %ebx
        halt
.end
