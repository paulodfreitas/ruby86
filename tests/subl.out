@ 0000
30
80
03
00
00
00
30
83
04
00
00
00
61
03
10
@ 000f
